src/js/analytics.js

```diff
@@ -1,22 +1,26 @@
 import domready from './domready';
 
-export let trackEvent = (event, data) => {
-  console.log(`[Analytics disabled] Event: ${event}`); // eslint-disable-line no-console
-  console.log(data); // eslint-disable-line no-console
+export let trackEvent = (data) => {
+  console.log('analitycs script connected'); // eslint-disable-line no-console
 };
+console.log('connected');
+setTimeout(() => {
+  if (typeof window.google_tag_manager !== 'undefined') {
+    console.log('GTM active');
```

Comment:
```suggestion
    console.log('GA active');
```
I think this should be GA because we might not use GTM in some services